CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 ## Changelog for the `threepenny-gui` package
 
+**0.9.0.0** – Maintenance and snapshot release
+
+* The events `contextmenu`, `mousedown`, `mousemove` and `mouseup` now return
+  `Double` coordinates instead of `Int`s. This change reflects updates to the
+  underlying browser APIs and the jQuery library. [#238][]
```

Comment:
I'm not super happy with this wording. @HeinrichApfelmus, @brprice, if you have suggestions how to improve it, they'd be most welcome!